containers/groovy-examples-redpanda/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.4"
+
+services:
+  deephaven:
+    image: ghcr.io/deephaven/server-slim:${VERSION:-latest}
+    ports:
+      - "${DEEPHAVEN_PORT:-10000}:10000"
+    volumes:
+      - ./data:/data
+    environment:
+      - START_OPTS=-Xmx4g
+
+  examples:
+    image: ghcr.io/deephaven/examples
+    volumes:
+      - ./data:/data
+    command: initialize
+
+  redpanda:
+    command:
+    - redpanda
+    - start
+    - --smp
+    - '1'
+    - --reserve-memory
+    - 0M
+    - --overprovisioned
+    - --node-id
+    - '0'
+    - --kafka-addr
+    - PLAINTEXT://0.0.0.0:29092,OUTSIDE://0.0.0.0:9092
+    - --advertise-kafka-addr
+    - PLAINTEXT://redpanda:29092,OUTSIDE://localhost:9092
+    image: docker.redpanda.com/vectorized/redpanda:v22.2.2
+    ports:
+    - 9092:9092
+    - 29092:29092
+
+  registry:
+    image: apicurio/apicurio-registry-mem:1.2.2.Final
```

Comment:
We should update our code - we shouldn't need apicurio registry; it's built into redpanda.